packages/shared/src/components/Feed.spec.tsx

```diff
@@ -410,7 +410,7 @@ it('should send add bookmark mutation', async () => {
   await waitFor(() => expect(mutationCalled).toBeTruthy());
 });
 
-it('should send remove bookmark mutation', async () => {
+it('should send remove bookmark mutation (feed)', async () => {
```

Comment:
Better way is to use describe as we do in newer files and then in test results you can see under which one it ran.